### PR TITLE
[Docs] Update Java Low-Level documentation to reflect shaded deps

### PR DESCRIPTION
**Pull request by tlrx**
This PR updates the documentation of the Java Low-Level Rest Client to reflect #25208.

### CODE REVIEW

**Review by hub-cap — APPROVED** [review]
Thanks a lot for fixing this

File: docs/java-rest/low-level/index.asciidoc [hunk 1]
```diff
@@ -22,6 +20,8 @@ The low-level client's features include:
 
 * optional automatic <<sniffer,discovery of cluster nodes>>
 
+* packaged as a single JAR file that includes all required dependencies
```
Comment (nik9000) [inline docs/java-rest/low-level/index.asciidoc]: s/includes/shades/?

s/required//?

**Review by nik9000 — APPROVED** [review]

**Review by javanna — APPROVED** [review]
thanks a lot @tlrx

**Comment by tlrx** [comment]
Thanks @hub-cap @nik9000 @javanna

**Pull request merged**